packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,165 @@
+
+When users select the footnote, you can display a popover that provides more information than would fit in the footnote:
+
+<div class="ws-docs-content-img">
+![Selected footnote, with an informative popover.](./img/footnote.svg)
+</div>
+
+## Usage
+
+When ChatBots are designed to meet the needs of your users, they can improve the overall UX of your product by offering convenient, efficient, and persistent support. When your ChatBot cannot find an answer for your users, you must provide them with a method to contact human support.
+
+Before building a ChatBot, make sure that you have justified it as an appropriate solution by asking yourself these questions:
+- What are the users’ goals?
+- How in-depth is the assistance the user will need?
+- Does human assistance better serve your users?
+- How is a ChatBot superior to online documentation, contextual support or wizards?
+- What data sources or abilities can this ChatBot leverage to assist your users?
+
+Do not create a ChatBot simply for the sake of novelty.
+
+### When to use a ChatBot 
+
+Use a ChatBot to offer your users on-demand help at any time, including:
+- Technical support and troubleshooting.
+- Product information and documentation.
+- Sales and product recommendations.
+- Training and onboarding.
+- System monitoring and alerts.
+- Community engagement .
+- Feedback collection and surveys.
+
+### When not to use a ChatBot
+
+Do not use a ChatBot when: 
+- A task could be accomplished more efficiently through the UI. 
+- A process is very complex or could take a long time.
+- A real human is needed for sensitive or emotional topics.
+
+## Behavior 
+
+### Accessing a ChatBot
+
+Users can enter a conversation with a ChatBot by clicking on the toggle. Once the ChatBot window opens, the toggle will change to display an "angle down" icon to indicate that clicking the toggle again will minimize the ChatBot. Users can select the toggle at any point in their journey to open and close the ChatBot as needed.
+
+<div class="ws-docs-content-img">
+![2 toggles with open and closed icons.](./img/chatbot-toggle.svg)
+</div>
+
+When there is an unread message from the ChatBot, a notification badge should be placed on the toggle.
+
+<div class="ws-docs-content-img">
+![Toggle with notification badge.](./img/chatbot-toggle-notification.svg)
+</div>
+
+### Sending messages and attaching files
+
+To message the ChatBot, users can type directly into the message bar in the footer or click any included actions. 
+
+<div class="ws-docs-content-img">
+![Elements of a message bar, including input actions.](./img/message-bar-elements.svg)
+</div>
+
+1. **Attach button:** Allows users to upload files from their computer. 
+1. **Use microphone button:** Supports speech recognition to allow users to use voice input.
+1. **Send button:** Allows users to send a typed message. This button should be disabled until a user has input text.
+
```

Comment:
I think we should add a 4th element here to show the attach menu. The attach menu component appears when the user clicks on the attachment button. The menu provides the user with various methods to upload content to the chatbot. You can change the behavior of the attach button to open a menu, rather than the default file viewer for your operating system. 
![attach-menu](https://github.com/user-attachments/assets/abc50a82-c344-4686-a77f-8aae1789ff32)

Reply:
The user has to provide whatever goes in that dropdown just fyi - we just provide the dropdown (drop-up in this case) and search bar.